special-interest-groups/sig-docs/member-list.md

```diff
@@ -312,3 +310,8 @@
 - [Reminiscent](https://github.com/Reminiscent)
 - [xiaoyangwang5](https://github.com/xiaoyangwang5)
 - [yuzhibotao](https://github.com/yuzhibotao)
+- [Anastasia-Lu](https://github.com/Anastasia-Lu)
+- [HuangPingZhong](https://github.com/HuangPingZhong)
+- [huangxiuyan](https://github.com/huangxiuyan)
+- [kolbe](https://github.com/kolbe)
+- [niezefeng](https://github.com/niezefeng)
```

Comment:
I didn't see merged PR in four doc repos by this potential contributor. Please help confirm.